src/vm/i386/unixstubs.cpp

```diff
@@ -6,7 +6,11 @@
 
 extern "C"
 {
-    void ThrowControlForThread()
+    void ThrowControlForThread(
+#ifdef WIN64EXCEPTIONS        
```

Comment:
Well, after we switch, there is no reason for leaving in things that are not needed.